src/Csrf/CsrfViewInjectionInterface.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Yii\View;
+namespace Yiisoft\Yii\View\Csrf;
 
 interface CsrfViewInjectionInterface
```

Comment:
Do we need this interface? There's a single implementation and no use of the interface.